lib/livebook/hubs/team.ex

```diff
@@ -4,27 +4,48 @@ defmodule Livebook.Hubs.Team do
   use Ecto.Schema
   import Ecto.Changeset
 
+  defmodule Offline do
+    @moduledoc false
+    use Ecto.Schema
+
+    alias Livebook.Secrets.Secret
+
+    @type t :: %__MODULE__{
+            secrets: list(Secret.t())
+          }
+
+    @primary_key false
+    embedded_schema do
+      field :secrets, {:array, :map}, default: []
+    end
+  end
+
   @type t :: %__MODULE__{
           id: String.t() | nil,
-          org_id: pos_integer() | nil,
-          user_id: pos_integer() | nil,
-          org_key_id: pos_integer() | nil,
-          teams_key: String.t() | nil,
-          org_public_key: String.t() | nil,
-          session_token: String.t() | nil,
+          org_id: non_neg_integer(),
+          user_id: non_neg_integer(),
+          org_key_id: non_neg_integer(),
+          teams_key: String.t(),
+          org_public_key: String.t(),
+          session_token: String.t(),
           hub_name: String.t() | nil,
-          hub_emoji: String.t() | nil
+          hub_emoji: String.t() | nil,
+          offline: Offline.t() | nil
         }
 
+  @enforce_keys [:user_id, :org_id, :org_key_id, :session_token, :org_public_key, :teams_key]
+
   embedded_schema do
-    field :org_id, :integer
-    field :user_id, :integer
-    field :org_key_id, :integer
-    field :teams_key, :string
-    field :org_public_key, :string
-    field :session_token, :string
+    field :org_id, :integer, default: 0
+    field :user_id, :integer, default: 0
+    field :org_key_id, :integer, default: 0
+    field :teams_key, :string, default: ""
+    field :org_public_key, :string, default: ""
+    field :session_token, :string, default: ""
```

Comment:
Sorry, let's not set the default here. Instead, let's set them when we create the hub (which you already do). If we also set the default, it may be harder to find bugs later.